Make.inc

```diff
@@ -1002,9 +1002,16 @@ MKLLIB := $(MKLROOT)/lib/ia32
 endif
 USE_SYSTEM_BLAS:=1
 USE_SYSTEM_LAPACK:=1
+ifeq ($(OS), WINNT)
+LIBBLASNAME := mkl_rt
+LIBLAPACKNAME := mkl_rt
+# work around libtool issue with arpack
+MKL_LDFLAGS := -L$(MKLLIB) -Wl,-lmkl_rt
```

Comment:
The issue is that libtool isn't smart enough to find and count a dll that doesn't start with "lib" if you don't also have a .dll.a (or maybe .lib) import library available for it. This `-Wl,-lmkl_rt` tells libtool to just send this through to the linker unmolested without libtool doing its interception and modification of build commands.